### PR TITLE
Fixes for reallydown

### PR DESCRIPTION
**Pull request by spladug**
:eyeglasses: @rram @kjoconnor @umbrae

### CODE REVIEW

File: 504.reallydown.html [hunk 1]
```diff
@@ -153,15 +153,12 @@
 	'sorry, europe',
 	'you could always go outside or something',
 	'time actually is going slower right now',
-	'did you see today\'s <a href="//xkcd.com">xkcd</a>?',
 	'go ahead, click refresh again.',
-	'there may be more information in <a href="https://twitter.com/redditstatus">@redditstatus</a>',
-	'did you know we\'re open source? <a href="https://github.com/reddit/">see if you can find the bug</a>.',
```
Comment (rram) [inline 504.reallydown.html]: 🔥

File: 504.reallydown.html [hunk 2]
```diff
@@ -153,15 +153,12 @@
...
-	'i wonder if you\'ll have a <img src="//www.redditstatic.com/mail.png" alt="orangered"> when reddit\'s back?',
+	'there may be more information on <a href="http://www.redditstatus.com/">redditstatus.com</a>',
+	'i wonder if you\'ll have an <img src="//www.redditstatic.com/mail.png" alt="orangered"> when reddit\'s back?',
 	'in the meantime, go check out what\'s new at <a href="http://redditgifts.com">redditgifts</a>!',
 	'it\'s not you, it\'s me.',
 	'I wonder what\'s new on reddi-- oh, sorry.',
-	'this episode was brought to you by the letter ctrl-Z',
 	'don\'t touch that dial!',
 	'well, there goes our new years resolutions.',
 	'BSOD: blue screen of downtime',
```
Comment (rram) [inline 504.reallydown.html]: 🔕 I feel like there needs to be a "turning it off and on again" joke here

**Comment by rram** [comment]
🐟

File: 504.reallydown.html (hunk 2 above)
Comment (spladug) [inline 504.reallydown.html]: pull requests gladly accepted :P

**Comment by umbrae** [comment]
🐟

**Comment by dmuth** [comment]
👍

**Pull request merged**